backend/internal/cmds/mgmt/serve/connect/cmd.go

```diff
@@ -410,9 +411,16 @@ func serve(ctx context.Context) error {
 	}
 	mux.Handle("/", api)
 
+	c := cors.New(cors.Options{
```

Comment:
TODO: Need to test this with same domains. May only need to allow this for development as in Staging, app and api are under the same root domain